### PR TITLE
Pull request for new site 

### PR DESCRIPTION
**Pull request by Sartish**

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
Hi Sara! Do you have a link to your deployed project that I can have a look at? :)

File: code/index.html [hunk 1]
```diff
@@ -2,21 +2,245 @@
...
     <header>
       <!-- Some kind of logo and navbar -->
+      <nav> 
+        <div class="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+        <img class="logo" src="Penguin_logo.svg" alt="">
```
Comment (HIPPIEKICK) [inline code/index.html:20]: Don't omit the `alt` attribute ;)

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,245 @@
...
+
+        <div class="right-nav">
+          <a href="About">About</a>  
+          <a href="Contact">Contact</a>  
+          <button a href="Get involved">Sign up</a> </button>
```
Comment (HIPPIEKICK) [inline code/index.html:25]: Missing tag alert!
```
<button>
  <a>
  </a>
</button>
```
Or even better - styling the `a` tag to look like a button?

File: code/index.html [hunk 3]
```diff
@@ -2,21 +2,245 @@
...
+      <img src="./lion.png" alt="">
+      <div class=head-line>
+        <div class="slogan">
+          <h1>SAVE YOUR PLANET TODAY!</h1>
+          <p>Yes. It is still possible</p>
+        </div>
+        <div class="buttons">
+        <button>Read more</button>
+        </div>
+        </div>
```
Comment (HIPPIEKICK) [inline code/index.html:44]: Indent for every nested element:
```
<div class=head-line>
  <div class="slogan">
    <h1>SAVE YOUR PLANET TODAY!</h1>
    <p>Yes. It is still possible</p>
  </div>
  <div class="buttons">
    <button>Read more</button>
  </div>
</div>
```

File: code/index.html [hunk 4]
```diff
@@ -2,21 +2,245 @@
...
+
+<!-- SMALL NEWS  -->
+
+    <div class="small-news-heading">
+      <h2>Our latest news</h2>
+    </div>
+
+    <section class="small-news">
+      <div class="box-one">
+          <div class=photo>
```
Comment (HIPPIEKICK) [inline code/index.html:67]: Don't forget the quotation marks around the class name

File: code/index.html [hunk 5]
```diff
@@ -2,21 +2,245 @@
...
+      <p>Twitter</p>
+      <p>Facebook</p>
+      <p>Instagram</p>
+    </div>
+    <div class=info-three>
+      <p>how it works</p>
+      <p>Terms of conditions</p>
+      <p>Tell me more</p>
+    </div>
+  </footer>
```
Comment (HIPPIEKICK) [inline code/index.html:244]: Nice with a footer!

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,446 @@
...
+}
+/*******DEFAULT SETTING*****/
+button{
+  padding: 10px; 
+  border:0.16em solid #002e63;
+  background:#002e63;
+  color:white;
+  text-align:center;
+  transition: all 0.20s; /*makes the a smooth "alower" effect on hover*/
+    }
```
Comment (HIPPIEKICK) [inline code/style.css:15]: Nice button! `cursor: pointer;` is nice as well :)

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,446 @@
...
+
+  button:hover{
+  color:#DDDDDD;
+  border-color:#DDDDDD;
+    }
+
+a{
+  text-decoration: none;
+  color:black;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:31]: Indentation is a little off here, the closing tag should not be indented (and there should be a space between the element and the opening tag, _and_ between property key and value):

```
button:hover {
  color: #DDDDDD
  border-color: #DDDDDD
}
```

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,446 @@
...
+
+
+.big-news img{
+  display: flex;
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+
+}
```
Comment (HIPPIEKICK) [inline code/style.css:88]: Take a look at the properties object-fit and object-position to get rid of the distortion!

File: code/style.css [hunk 9]
```diff
@@ -1,4 +1,446 @@
...
+.small-news-heading h2{
+  margin-left: 3%;
+  }
+
+.box-one {
+  display: flex;
+  flex-direction: column; 
+  padding: 20px;
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:173]: Love this styling!

File: code/style.css [hunk 10]
```diff
@@ -1,4 +1,446 @@
...
+.projects .over:hover {
+  opacity: 1;
+}
+
+.projects .over .about-text {
+  margin-top: 100px;
+  opacity: 1;
+  transition-delay: 0s;
+  transition-duration: 0.4s;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:292]: Wow!

File: code/style.css [hunk 11]
```diff
@@ -1,4 +1,446 @@
...
+    justify-content: space-between;
+  }
+
+  .head-line
+  {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:391]: Opening curly bracket should be on the same line as the selector.

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Good job with this project Sara! You managed well with the flexboxing! I would just like to ask you to take another look at your formatting before approval (HTML: indentation, giving nested elements new lines. CSS: space between element and opening curly bracket, indentation of rules inside curly brackets, space between elements...) Take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap :)

We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/index.html [hunk 12]
```diff
@@ -2,21 +2,246 @@
...
+          </div>
+        <div class="minihead-line">
+            <a href="https://www.nationalgeographic.com/magazine/2018/11/antarctica-climate-change-western-peninsula-ice-melt-krill-penguin-leopard-seal/">
+              <div class="text">
+              <h3>The big meltdown</h3>
+              <p>Scientist are alarmed by the rapid changes</p>  
+            </a>
+          </div>
+        </div>
+    </div>
```
Comment (HIPPIEKICK) [inline code/index.html:74]: Hi Sara! Still some indentation changes that need to be done. Here is an example:

```
<div class="box-one">
  <div class="photo">
    <img src="./antartica.png" alt="antartica">
  </div>
  <div class="minihead-line">
    <a href="https://www.nationalgeographic.com/magazine/2018/11/antarctica-climate-change-western-peninsula-ice-melt-krill-penguin-leopard-seal/">
      <div class="text">
        <h3>The big meltdown</h3>
        <p>Scientist are alarmed by the rapid changes</p>  
      </div>
    </a>
  </div>
</div>
```

Comment (Sartish) [reply code/index.html]: Omg Can't believe I am so bad at indentations. I tried now look through each line and used format HTML language.  For me now it looks good, but maybe I missed something🙈

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**